### PR TITLE
recommended products show on user profile

### LINKED ISSUE #1: Add recommended products to user profile
Currently, when `/profile` is requested from the API, the products that the user has recommended to other users is included in the response.

```json
"recommends": [
    {
        "product": {
            "id": 32,
            "name": "DB9"
        },
        "customer": {
            "id": 5,
            "user": {
                "first_name": "Joe",
                "last_name": "Shepherd",
                "email": "joe@joeshepherd.com"
            }
        }
    }
]
```

This response needs to be augmented to also include the products that were recommended by another user to this user. 

1. The property should be `recommended`
2. It should be an array of recommendations that were made to the current user, not by them

```json
"recommendations": [
    {
        "product": {
            "id": 50,
            "name": "Escalade EXT"
        },
        "recommender": {
            "id": 7,
            "user": {
                "first_name": "Brenda",
                "last_name": "Long",
                "email": "brenda@brendalong.com"
            }
        }
    }
]
```

### LINKED ISSUE #22: Bug: Same user profile always returned
When an authenticated client submits a GET request to the `/profile` resource, an incorrect user profile is returned. Additionally, the same user profile is returned regardless of the authentication token provided.

### PR DESCRIPTION
**Pull request by kritikillz931**
Description of PR that completes issue here...
Products recommended by other users are now included in current users profile data
## Changes

- seeded and migrated database with existing json fixtures.
- created a recommendations fixture and loaded data to database
- added recommends & recommendations QuerySet to profile.py List try clause
- added RecommendationSerializer to profile.py with recommender and product variables
- added recommendations variable to ProfileSerializer in profile.py 
- added 'recommendations' to the class meta fields on ProfileSerializer

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

GET http://localhost:8000/profile 

GETS current user profile with products they've recommended and what others have recommended to them

**Response**

```json
{
    "id": 4,
    "url": "http://localhost:8000/customers/4",
    "user": {
        "first_name": "Steve",
        "last_name": "Brownlee",
        "email": "steve@stevebrownlee.com"
    },
    "phone_number": "555-1212",
    "address": "100 Infinity Way",
    "payment_types": [],
    "recommends": [
        {
            "product": {
                "id": 4,
                "name": "H1"
            },
            "customer": {
                "id": 6,
                "user": {
                    "first_name": "Jisie",
                    "last_name": "David",
                    "email": "jisie@jisiedavid.com"
                }
            }
        }
    ],
    "recommendations": [
        {
            "product": {
                "id": 10,
                "name": "Land Cruiser"
            },
            "customer": {
                "id": 4,
                "user": {
                    "first_name": "Steve",
                    "last_name": "Brownlee",
                    "email": "steve@stevebrownlee.com"
                }
            }
        }
    ]
}
```

HTTP/1.1 200 OK

## Related Issues

- Fixes #1 
- Fixes #22

### CODE REVIEW

File: bangazonapi/views/profile.py [hunk 1]
```diff
@@ -81,8 +81,11 @@ def list(self, request):
             }
         """
         try:
-            current_user = Customer.objects.get(user=4)
+            current_user = Customer.objects.get(user=request.auth.user)
```
Comment (hannahhall) [inline bangazonapi/views/profile.py:84]: Can you add a note to the pr description that this pr also fixes ticket #22?

**Review by hannahhall — CHANGES_REQUESTED** [review]
Your code looks good but you pushed up your node_modules folder. Delete that folder then push up again

File: bangazonapi/views/profile.py (hunk 1 above)
Comment (kritikillz931) [reply bangazonapi/views/profile.py:84]: added those changes and re pushed

**Review by hannahhall — APPROVED** [review]
👍

**Pull request merged**